index.html

```diff
@@ -135,6 +135,34 @@ <h4 class="molo">Agrios is the largest global organic farm.</h4>
       
     </section>
     </section>
+
+    <!-- lilly section -->
+    
+ <section>
+  <section>
```

Comment:
Great job so far, observe the following that your section defies the class `.general-spacing` which means your section doesn't obey the general padding added to other sections. Attempt to remove it using negative spacing.

Target this containers and use a code similar to the snippet below:

<img width="844" alt="image" src="https://github.com/user-attachments/assets/f908f6c6-3275-4857-bff0-6d7761052a9a">

Play with it until you find a suitable value. You may also need to work on width a bit to get closer to the design.

Reply:
Thank you so much for the review and suggestions @brytebee 
i have worked on the suggested changes, by adding a class=.general-spacing to my <section> 
But have faced challenges with loading the attached image